container/build.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash -e
 BRANCH=master
 IMAGE=opencontrail/developer-sandbox
+DOCKERFILE=Dockerfile.centos
 
-while getopts ":b:i:" opt; do
+while getopts ":d:b:i:" opt; do
```

Comment:
not so obvious that I need to pass flag 'd' to build rhel :)
but I have no idea what's better...

Reply:
ah, got it. it's a name for dockerfile... then maybe it's better to pass something like os_family or similar name that is used to differentiate rhel/centos ?

Reply:
it's a single character option... I can change the character name or add a comment

Reply:
I meant that we can pass os_family and use it as extension for Dockerfile instead of passing full name of Dockerfile

Reply:
I'd prefere to pass just centos or rhel as a parameter.  And then use it inside lilke : Dockerfile.$LINUX_DISTR

(I'd prefere to use the same semantic as in contrail-container-builder  - there is a LINUX_DISTR which can be centos or rhel for now,  and there is LINUX_DISTR_VER which can be latest or exact version like 7.4.1708)